example/.cargo/config.toml

```
@@ -0,0 +1,2 @@
+[alias]
+xtask-kompari = "run --package xtask_kompari --"
```

I think it would be a better UX if the name is just `xtask`. That would mean that in each repository, we get the same commands (`cargo xtask test`, etc.)
See e.g. https://github.com/gfx-rs/wgpu/blob/trunk/.cargo/config.toml. I think for now, we can probably ignore https://github.com/gfx-rs/wgpu/pull/3844#issuecomment-1579422557

```suggestion
xtask = "run --package xtask_kompari --"
```

---

Done